client/lib/olark/index.js

```diff
@@ -164,6 +186,36 @@ const olark = {
 		}
 	},
 
+	syncStoreWithExpandedState() {
+		// We query the dom here because there is no other 100% accurate way to figure this out. Olark does not
+		// provide initial events for api.box.onExpand when the api.box.show event is fired.
+		const isOlarkExpanded = !! document.querySelector( '.olrk-state-expanded' );
+		if ( isOlarkExpanded !== olarkStore.get().isOlarkExpanded ) {
+			olarkActions.setExpanded( isOlarkExpanded );
+		}
+	},
+
+	hookExpansionEventToStoreSync( eventName ) {
+		olarkEvents.on( eventName, this.syncStoreWithExpandedState );
+	},
+
+	showAvailabilityNotice() {
+		const noticeOptions = { showDismiss: true };
+		const { isOperatorAvailable, isUserEligible, isOlarkExpanded, isOlarkReady } = olarkStore.get();
+		const onEligibleContactForm = isUserEligible && window.location.pathname === '/help/contact';
```

Comment:
This is not a blocker now, but I find it odd to have this tied to a url from here, seems like it the route should choose to display it not the module look at the route.